src/decorators/any.ts

```diff
@@ -85,13 +83,12 @@ export interface AnySchemaModifiers {
     /**
      * Annotates the key
      */
-    notes(notes: string | string[]): this;
+    note(notes: string | string[]): this;
 
     /**
      * Annotates the key
      */
-    tags(tag: string, ...tags: string[]): this;
-    tags(tags: string | string[]): this;
+    tag(tag: string | string[], ...tags: string[]): this;
```

Comment:
This might sound like a nit but I think the original signature with the overload is still the most accurate. Can we please leave it as:

```typescript
    tags(tag: string, ...tags: string[]): this;	    tag(tag: string | string[], ...tags: string[]): this;
    tags(tags: string | string[]): this;
```

Reply:
My bad, it's fixed.